docs/user/discover.asciidoc

```diff
@@ -238,6 +242,8 @@ the document table, the sort order, and the {data-source}.
 
 . In the toolbar, click **Save**.
 
+. Optionally store <<managing-tags,tags>> and the time range with the search.
+
```

Comment:
Would be also great to update the screenshot in Visualize section below

<img width="300" alt="Screenshot 2022-09-30 at 09 25 17" src="https://user-images.githubusercontent.com/1415710/193215099-4b5b2cf5-c0ce-45d4-a8a1-42ee105d1d49.png">

Reply:
Done!